README.md

```diff
@@ -37,7 +37,7 @@ function onShutdown () {
 function healthCheck () {
   return Promise.resolve(
     // optionally include a resolve value to be included as
-    // info in the healthcheck response
```

Comment:
After some research, in the programming community, `healthcheck` seems more popular. Said, this I'd leave this unchanged, but I'd change `function healthCheck ()` to `function healthcheck ()`.
@silasbw thoughts?

Reply:
Let's do what's consistent with this code base and the other Node.js projects on @godaddy: "health check" is two words, so associated variable names will be camel cased as "healthCheck".